docs/rules/group-exports.md

```diff
@@ -0,0 +1,62 @@
+# group-exports
+
+Reports when multiple named exports or CommonJS assignments are present in a single file and when the default export is not adjacent to the named export.
```

Comment:
Checks about grouping named and default exports next to each other have been removed.